{"name": "testcaseadd3[【类型】输入空格，给出提示信息1]", "status": "passed", "steps": [{"name": "步骤一： 和3.14开始计算，预期应等于参数大小超出范围，打印<ExceptionInfo for raises contextmanager>", "status": "broken", "statusDetails": {"message": "TypeError: '>' not supported between instances of 'str' and 'int'\n", "trace": "  File \"D:\\zj\\PycharmProjects\\study202208\\practice\\study\\study0824\\testcase\\test_add.py\", line 48, in testcaseadd3\n    result = self.cal.add(x,y)\n  File \"D:\\zj\\PycharmProjects\\study202208\\script\\calculator.py\", line 4, in add\n    if a > 99 or a < -99 or b > 99 or b < -99:\n"}, "start": 1661497761902, "stop": 1661497761902}], "attachments": [{"name": "log", "source": "f4f59a30-ca92-46d3-91f3-f7e2830b03b4-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "x", "value": "' '"}, {"name": "y", "value": "3.14"}, {"name": "expect", "value": "'参数大小超出范围'"}], "start": 1661497761901, "stop": 1661497761902, "uuid": "2afae84e-88ec-4db7-af5d-43f6e99873b1", "historyId": "2802b86a58dfbbd8382ab8487235bceb", "testCaseId": "2f8f937ca82ff540c8b5f46a0207716d", "fullName": "practice.study.study0824.testcase.test_add.Testadd#testcaseadd3", "labels": [{"name": "story", "value": "加法模块测试error"}, {"name": "tag", "value": "P2"}, {"name": "parentSuite", "value": "practice.study.study0824.testcase"}, {"name": "suite", "value": "test_add"}, {"name": "subSuite", "value": "Testadd"}, {"name": "host", "value": "LAPTOP-BEO6NH7O"}, {"name": "thread", "value": "33120-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "practice.study.study0824.testcase.test_add"}]}